src/lamp_py/runtime_utils/remote_files.py

```diff
@@ -166,6 +167,14 @@ def s3_uri(self) -> str:
 tableau_bus_recent = S3Location(bucket=S3_PUBLIC, prefix=os.path.join(TABLEAU, "bus", "LAMP_RECENT_Bus_Events.parquet"))
 tableau_bus_all = S3Location(bucket=S3_PUBLIC, prefix=os.path.join(TABLEAU, "bus", "LAMP_ALL_Bus_Events.parquet"))
 
+# archive to archive_pii
+tableau_bus_operator_mapping_recent = S3Location(
+    bucket=S3_ARCHIVE, prefix=os.path.join(TABLEAU, "bus", "LAMP_RECENT_Bus_Operator_Mapping.parquet")
```

Comment:
separating these to archive is good for now but we should think about where we want to put this long-term and how we want folks within the T to access it. I don't think many folks need to be able to map these values themselves. We can address that next week tho for [this ticket](https://app.asana.com/1/15492006741476/project/1189492770004753/task/1211757434160344)